src/leetCodeExecutor.ts

```diff
@@ -48,10 +48,11 @@ class LeetCodeExecutor {
             }
             return false;
         }
-        try { // Check company plugin
-            await this.executeCommandEx("node", [await this.getLeetCodeBinaryPath(), "plugin", "-e", "company"]);
-        } catch (error) { // Download company plugin and activate
-            await this.executeCommandEx("node", [await this.getLeetCodeBinaryPath(), "plugin", "-i", "company"]);
+        for (const plugin of ["company", "solution.discuss", "leetcode.cn"]) { // Make sure plugin exists
```

Comment:
I remember `-i` will automatically enable it, is that right?

Reply:
The `getLeetCodeEndpoint` returns the configuration from vscode. No matter `leetcode.cn` is enabled or not, the vscode configuration will rectify it.

Reply:
Yes, it's because of the cache.

Still cannot see the purpose to change the logic here, the original implementation looks fine to me.

Reply:
For unknown reason, when  I debug the extension on my Linux machine, it pops out `PLUGIN NOT FOUND` on the execution of `"node", [await this.getLeetCodeBinaryPath(), "plugin", "-e", "leetcode.cn"]`, rather than download it and install.  But after I install the plugin, everything works fine again.

Reply:
Is this reproducible on your side?

Reply:
After installed `leetcode.cn.js`, I tried to delete it, then debug the extension, but the bug can no longer be produced anymore. I guess it can only be encountered at the very first time of using `leetcode-cli`.

Reply:
I suggest we leave the code as it's original implementation. If this is a problem and users open issues for it, we can address it in another PR?

Reply:
This could also be an error accidentally thrown by executing `leetcode plugin -i xx`, who knows.

Reply:
Reasonable suggestion, updated